stable/ambassador/templates/rbac.yaml

```diff
@@ -29,6 +29,13 @@ rules:
   - apiGroups: [ "apiextensions.k8s.io" ]
     resources: [ "customresourcedefinitions" ]
     verbs: ["get", "list", "watch"]
+  {{- if .Values.podSecurityPolicies }}
```

Comment:
Perhaps better to put under rbac.podSecurityPolicies
So it doesn't confuse if we decide to ship a psp with values like podSecurityPolicy.enabled :)

Reply:
Sounds good